public/app/plugins/datasource/cloudwatch/datasource.js

```diff
@@ -379,7 +389,7 @@ function (angular, _, moment, dateMath, kbn, CloudWatchAnnotationQuery) {
 
         if (dimensionKey) {
           var variable = _.find(templateSrv.variables, function(variable) {
-            return templateSrv.containsVariable(target.dimensions[dimensionKey], variable.name);
+            return self.containsVariable(target.dimensions[dimensionKey], variable.name);
```

Comment:
templateSrv does not contain the function containsVariable any more. Which means the current master does not work with muliple selected values.